README.md

```diff
@@ -19,7 +19,9 @@ npm install -g @asyncapi/generator
 
 |Name|Description|Required|Default|Allowed values|Example|
 |---|---|---|---|---|---|
+|frontMatter|The name of a JSON or YAML formatted file containing values to provide the YAML frontmatter for static-site or documentation generators. The file may contain {{title}} and {{version}} replaceable tags.|No|None|Any JSON or YAML formatted file|`slate.yaml`|
```

Comment:
we need more description here:
- please add links to those `slate` and `ssg` examples 
- explain where `title` and `version` is taken from, that it is `info.title` and `info.version` from the asyncapi document, but in case of the version, it can also be a value taken from `version` parameter

Reply:
maybe:

```suggestion
|frontMatter|The name of a JSON or YAML formatted file containing values to provide the YAML frontmatter for static-site or documentation generators, just like [this](test/spec/ssg.yaml) one. The file may contain {{title}} and {{version}} tags. They are replaced with information for the AsyncAPI document that is under `info.title` and `info.version`. You can overwrite the version with `version` parameter. [Here](test/spec/slate.yaml) you can see an example that uses tags in frontmatter compatible with [slate](https://github.com/Slatedocs/Slate) |No|None|Any JSON or YAML formatted file|`slate.yaml`|
```